app/Http/Kernel.php

```diff
@@ -45,7 +45,7 @@ class Kernel extends HttpKernel
 
         'api' => [
             EnsureFrontendRequestsAreStateful::class,
-            'throttle:60,1',
+            'throttle:200,1',
```

Comment:
This would mean a user could try 200 times per minute to login? Please add specific throttling for the login routes!

Reply:
No, I thought so too at first, but laravel has it's own login throttle so nothing to worry about https://laravel.com/docs/7.x/authentication#login-throttling